nf_core/modules/install.py

```diff
@@ -130,6 +130,14 @@ def install(self, module):
         if not self.download_module_file(module, version, self.modules_repo, install_folder):
             return False
 
+        # Print include statement
+        if module == "stringtie/stringtie":
+            # Only with stringtie the process name is STRINGTIE instead of STRINGTIE_STRINGTIE
```

Comment:
This is a bug. We should update the nf-core module to use `STRINGTIE_STRINGTIE`.

Reply:
I see. I can modify the module, but we should update all the pipelines that use that module then, right?

Reply:
Modifying just the module is great. When developers re-install the module they can update the pipeline code themselves.

Reply:
Update the module here https://github.com/nf-core/modules/pull/1546 @asthara10 if you fancy reviewing?